sysutils/mise/Portfile

```diff
@@ -19,15 +19,17 @@ long_description    mise is a front-end to your dev environment. \
 categories          sysutils
 license             MIT
 
-maintainers         {outlook.com:gjq.uoiai @MisLink} openmaintainer
+maintainers         {outlook.com:gjq.uoiai @MisLink} \
+                    {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
 
 checksums           ${distname}${extract.suffix} \
-                    rmd160  376e4eeca4c6538ce8f88f0d18c7e320e8e7e68b \
-                    sha256  757f4e5a8196a58d50a813306cbd50c25d975a4ed0b87f8ec1188308757f5569 \
-                    size    3051593
+                    rmd160  fcb4197c3ecb42bc843ba4dd8c81acc760891769 \
+                    sha256  e86e277e8a136d331667fccb13b1d44fda320271c8956a7bb13c110857990529 \
+                    size    3059332
 
 destroot {
-    xinstall -m 755 \
+    xinstall -m 0755 \
         ${worksrcpath}/target/[cargo.rust_platform]/release/${name} \
         ${destroot}${prefix}/bin/
```

Comment:
Install the completions and the `mise.1` man page. Sadly, this suggestion *will not* cleanly apply because I can't figure out how to get it to cover multiple lines.

```suggestion
destroot {
    set release_dir ${worksrcpath}/target/[cargo.rust_platform]/release

    xinstall -m 0755 ${release_dir}/${name} ${destroot}${prefix}/bin/

    set bash_completions_dir share/bash-completion/completions
    set fish_completions_dir share/fish/vendor_completions.d
    set zsh_completions_dir share/zsh/site-functions

    xinstall -d ${destroot}${prefix}/${bash_completions_dir}
    xinstall ${release_dir}/${bash_completions_dir}/${name} \
        ${destroot}${prefix}/${bash_completions_dir}

    xinstall -d ${destroot}${prefix}/${fish_completions_dir}
    xinstall ${release_dir}/${fish_completions_dir}/${name}.fish \
        ${destroot}${prefix}/${fish_completions_dir}

    xinstall -d ${destroot}${prefix}/${zsh_completions_dir}
    xinstall ${release_dir}/${zsh_completions_dir}/_${name} \
        ${destroot}${prefix}/${zsh_completions_dir}

    foreach miseman [glob ${worksrcpath}/man/man1/*.1] {
      copy ${miseman} ${destroot}${prefix}/share/man/man1
    }
}
```